docs/types.rst

```diff
@@ -0,0 +1,506 @@
+.. highlight:: yaml
+
+.. _types:
+
+Types
+^^^^^
+
+Node Types
+==========
+
+.. cfy:node:: cloudify.azure.nodes.ResourceGroup
+
+This example shows adding additional parameters, and explicitly defining the azure_config::
+
+  resourcegroup:
+    type: cloudify.azure.nodes.ResourceGroup
+    properties:
+      name: {concat:[ { get_input: resource_prefix }, rg ] }
+      location: { get_input: location }
+      azure_config:
+        subscription_id: { get_input: subscription_id }
+        tenant_id: { get_input: tenant_id }
+        client_id: { get_input: client_id }
+        client_secret: { get_input: client_secret }
+
+
+.. cfy:node:: cloudify.azure.nodes.storage.StorageAccount
+
+This example shows adding additional parameters, and explicitly defining the azure_config::
```

Comment:
This example shows adding ~~additional~~ **more** parameters

**_Repeat wherever this line appears_**